src/ORM/FieldType/DBFloat.php

```diff
@@ -60,7 +58,7 @@ public function scaffoldFormField(?string $title = null, array $params = []): ?F
 
     public function nullValue(): ?int
```

Comment:
```suggestion
    public function nullValue(): ?float
```
Shouldn't it be a float?

Reply:
Yes :-)